integration-tests/src/test/java/com/datastax/oss/driver/mapper/ComputedIT.java

```diff
@@ -0,0 +1,306 @@
+    ComputedDao computedDao = mapper.computedDao(sessionRule.keyspace());
+    int key = keyProvider.incrementAndGet();
+
+    CqlSession session = sessionRule.session();
+    PreparedStatement preparedStatement =
+        session.prepare("INSERT INTO computed_entity (id, c_id, v) VALUES (?, ?, ?)");
+    BoundStatementBuilder builder = preparedStatement.boundStatementBuilder();
+    ComputedEntity entity = new ComputedEntity(key, 1, 2);
+    BoundStatement statement = computedDao.set(builder, entity).build();
+    session.execute(statement);
+
+    // retrieve values to ensure was successful.
+    ComputedEntity retrievedValue = computedDao.findById(key, 1);
+
+    assertThat(retrievedValue.getId()).isEqualTo(key);
+    assertThat(retrievedValue.getcId()).isEqualTo(1);
+    assertThat(retrievedValue.getV()).isEqualTo(2);
+  }
+
+  @Test
+  public void should_return_computed_values_in_GetEntity() {
+    ComputedDao computedDao = mapper.computedDao(sessionRule.keyspace());
+    int key = keyProvider.incrementAndGet();
+
+    long time = System.currentTimeMillis() - 1000;
+    ComputedEntity entity = new ComputedEntity(key, 1, 2);
+    computedDao.saveWithTime(entity, 3600, time);
+
+    CqlSession session = sessionRule.session();
+    // query with the computed values included.
+    // since we the mapper expects the result name to match the property name, we used aliasing
+    // here.
+    ResultSet result =
+        session.execute(
+            SimpleStatement.newInstance(
+                "select id, c_id, v, writetime(v) as writetime, ttl(v) as myttl from "
+                    + "computed_entity where "
+                    + "id=? and "
+                    + "c_id=? limit 1",
+                key,
+                1));
+    assertThat(result.getAvailableWithoutFetching()).isEqualTo(1);
+
+    ComputedEntity retrievedValue = computedDao.get(result.one());
+    assertThat(retrievedValue.getId()).isEqualTo(key);
+    assertThat(retrievedValue.getcId()).isEqualTo(1);
+    assertThat(retrievedValue.getV()).isEqualTo(2);
+
+    // these should be set
+    assertThat(retrievedValue.getTtl()).isEqualTo(3600);
+    assertThat(retrievedValue.getWritetime()).isEqualTo(time);
+  }
+
+  @Test
+  public void should_return_computed_values_in_query() {
+    ComputedDao computedDao = mapper.computedDao(sessionRule.keyspace());
+    int key = keyProvider.incrementAndGet();
+
+    long time = System.currentTimeMillis() - 1000;
+    ComputedEntity entity = new ComputedEntity(key, 1, 2);
+    computedDao.saveWithTime(entity, 3600, time);
+
+    ComputedEntity retrievedValue = computedDao.findByIdQuery(key, 1);
+    assertThat(retrievedValue.getId()).isEqualTo(key);
+    assertThat(retrievedValue.getcId()).isEqualTo(1);
+    assertThat(retrievedValue.getV()).isEqualTo(2);
+
+    // these should be set
+    assertThat(retrievedValue.getTtl()).isEqualTo(3600);
+    assertThat(retrievedValue.getWritetime()).isEqualTo(time);
+  }
```

Comment:
I think we are missing test case that does `@Computed("ttl(v)")` and `@CqlName("myttl")` but cql name is not present in results from the query

Reply:
You mean just to validate that an exception is thrown if the user provided query does not match the expectation of what the result alias should be?